pyproject.toml

```diff
@@ -44,7 +44,10 @@ pandas = "^1.0.1"
 tqdm = "^4.45.0"
 dask = {extras = ["complete"], version = "^2.14.0"}
 dash = "1.12.0"
-tensorflow = "^2.2.0"
+tensorflow = "^1.13.1"
```

Comment:
Why do we need to downgrade the version?

Reply:
So I have a problem regarding this. If I upgrade the version, I can't load my model the way I want. But if the version is not at least 2, the groceries.real.py file prints out errors....

Reply:
We have decided to have this repo using tensorflow 2.X. Can you make sure to update your tensorflow model so that it's compatible with this requirements?